prime-dotnet-webapi/Models/BusinessEvent.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace Prime.Models
+{
+    [Table("BusinessEvent")]
+    public class BusinessEvent : BaseAuditable
+    {
+        [Key]
+        public int? Id { get; set; }
```

Comment:
Don't think we should have nullable indexes any more

Reply:
oh yeah good catch